hx_lti_initializer/views.py

```diff
@@ -232,6 +253,31 @@ def edit_course(request, id):
         if form.is_valid():
             course = form.save()
             course.save()
+            # this removes an administrator if they were checked off the list
+            admins = course.course_admins.all()
+            selected_list = request.POST.getlist('select_existing_user') + request.POST['new_admin_list'].split(',')
+            for admin in admins:
+                if admin.user.username not in selected_list:
+                    course.course_admins.remove(admin)
+                else:
+                    selected_list.remove(admin.user.username)
+            course.save()
+
+            # this will create an item in the database so when the user
+            # that was just added as an admin logs in, they get added
+            # to the list of admins in the course.
+            if len(selected_list) > 0:
+                for name in selected_list:
+                    if str(name.strip()) != "":
+                        try:
+                            new_course_admin = LTICourseAdmin(
```

Comment:
@arthurian Same as above? Is it okay to user username here or should it search for something else?